src/components/page-container/test/PageContainer.test.jsx

```diff
@@ -64,4 +65,38 @@ describe('PageContainer', () => {
       expect(await screen.findByText('Test Text')).toBeTruthy();
     });
   });
+
+  test('Page container with BackToInstructor compo without PUBLIC PATH', async () => {
+    await act(async () => {
+      const cohorts = { cohorts: [Factory.build('cohort'), Factory.build('cohort')] };
+      const courseMetadata = Factory.build('courseMetadata');
+
+      getCohorts.mockImplementation(() => cohorts);
+      getCourseHomeCourseMetadata.mockImplementation(() => courseMetadata);
+
+      render(
+        <PageContainer>
+          <BackToInstructor />
+        </PageContainer>,
+      );
+      const linkEl = await screen.findByText('Back to Instructor Dashboard');
+      expect(linkEl.href).toEqual('http://localhost:18000/courses/test-course-id/instructor#view-course-info');
+    });
+  });
+  test('Page container with BackToInstructor compo with PUBLIC_PATH', async () => {
+    await act(async () => {
+      const cohorts = { cohorts: [Factory.build('cohort'), Factory.build('cohort')] };
+      const courseMetadata = Factory.build('courseMetadata', {}, { host: 'http://localhost:18000/communication-path' });
```

Comment:
>  As I actually try to understand the test more deeply, it occurs to me to ask: how does courseMetadata.host affect the path?  

So my understanding is that when factory builds courseMetedata object,  it will pass the `host` to the `tab obecjt` hence here  https://github.com/openedx/frontend-app-communications/blob/4493eb71fa4fdaef3f5c541cff2d9f95d91ec14c/src/components/page-container/data/__factories__/courseMetadata.factory.js#L19-L29 

Of which tab object would create the url using  `course-id`,  `path` and `host`. hence 

https://github.com/openedx/frontend-app-communications/blob/4493eb71fa4fdaef3f5c541cff2d9f95d91ec14c/src/components/page-container/data/__factories__/tab.factory.js#L12 


> I think the right way to test this is going to be by mocking getConfig. More on that on a separate comment 

Hum... I think probably the right way to test it, to involve react-router, or at least override location vlaue from `useLocation` hook, and assuming devs won't use the `window` object

Reply:
I will look into, I don't how I missed your upper comment which is more relevant. 

> You can then try to mock window.location.pathname to be something that contains a PUBLIC_PATH and something that doesn't, and check that both work 

My understanding, is that if the future a dev by mistake replaced course-id that is taken from params, by window.location, the test should fail, right?

Reply:
@arbrandes you can check the last commit https://github.com/openedx/frontend-app-communications/pull/127/commits/e251025a70c6644ba235bfecc992d62242abcd6f  in the second test, I have changed window.locatoin.pathname value and test for both, course url and window.locaiotn.pathname.

Reply:
You can remove the new tests from `PageContainer`, now.